omod/src/main/webapp/resources/scripts/app.js

```diff
@@ -0,0 +1,64 @@
+
+var app  = angular.module('Tag',['tagService','ngDialog','uicommons.common.error']);
+
+
+app.controller('tagCtrl',['$scope','TagService','ngDialog',
+    function ($scope,TagService,ngDialog) {
+    var personUuid;
+    $scope.init = function(personUuid) {
+      $scope.thisPersonUuid = personUuid;
```

Comment:
I'm a little confused, I see here you have a scoped variable named `thisPersonUuid` and you also declared a global one above named personUuid.